docs/source/release/v6.13.0/Framework/Algorithms/Bugfixes/39041.rst

```diff
@@ -0,0 +1 @@
+- Fixes a bug in :ref:`algm-LoadAndMerge` where output workspaces were stored in the ADS when set not to. This caused the leaking of temporary workspaces in algorithms that call :ref:`algm-LoadAndMerge` as a child. Legacy behaviour can be achieved by passing `StoreInADS=True` to the algorithm.
```

Comment:
This is only worth changing if you have to push anyway, but you need to use double backticks for monospace formatting in `.rst` files. (Also applies to the other release note)
```suggestion
- Fixes a bug in :ref:`algm-LoadAndMerge` where output workspaces were stored in the ADS when set not to. This caused the leaking of temporary workspaces in algorithms that call :ref:`algm-LoadAndMerge` as a child. Legacy behaviour can be achieved by passing ``StoreInADS=True`` to the algorithm.
```